addons/website/data/website_contactus_data.xml

```diff
@@ -0,0 +1,174 @@
+                                                <div class="form-group col-12 s_website_form_field s_website_form_custom s_website_form_required" data-type="text" data-name="Field">
+                                                    <div class="row s_col_no_resize s_col_no_bgcolor">
+                                                        <label class="col-form-label col-sm-auto s_website_form_label" style="width: 200px" for="contact6">
+                                                            <span class="s_website_form_label_content">Your Question</span>
+                                                        </label>
+                                                        <div class="col-sm">
+                                                            <textarea id="contact6" class="form-control s_website_form_input" name="Description" required=""></textarea>
+                                                        </div>
+                                                    </div>
+                                                </div>
+                                                <div class="form-group col-12 s_website_form_field s_website_form_dnone">
+                                                    <div class="row s_col_no_resize s_col_no_bgcolor">
+                                                        <label class="col-form-label col-sm-auto s_website_form_label" style="width: 200px" for="contact7">
+                                                            <span class="s_website_form_label_content">Email To</span>
+                                                        </label>
+                                                        <div class="col-sm">
+                                                            <input id="contact7" type="hidden" class="form-control s_website_form_input" name="email_to"/>
+                                                        </div>
+                                                    </div>
+                                                </div>
+                                                <div class="form-group col-12 s_website_form_submit" data-name="Submit Button">
+                                                    <div style="width: 200px;" class="s_website_form_label"/>
+                                                    <a href="#" role="button" class="btn btn-primary btn-lg s_website_form_send">Submit</a>
+                                                    <span id="s_website_form_result"></span>
+                                                </div>
+                                            </div>
+                                        </form>
+                                    </div>
+                                </section>
+                            </div>
+                            <div class="col-lg-4 mt-4 mt-lg-0">
+                                <address itemscope="itemscope" itemtype="http://schema.org/Organization">
```

Comment:
`<address>` is now useless since we don't use t-fields + "widget": "contact" anymore, which were in charge of adding the itemprop :/
So either we remove the `<address>` schema tag or we add the itemprop in the DOM (including on phone and on mail, but those are not supposed to be inside address tag AFAIK so it seems it was wrong before).

One problem would be that the enduser would mess with those hardcoded itemprop, eg replace the phone by the email and now the itemprop phone contains an email..

So I'd vote to simply remove the `<address>` since it is not a t-field anymore and it makes sense only for a t-field as we know what will be written inside.

Anyone else's opinion is welcome here
@JKE-be @sbuhl